nuxeo-coldstorage-web/ftest/features/coldstorage.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Cold Storage
+
+  Background:
+    Given user "Jack" exists in group "members"
+
+  Scenario: Give WriteColdStorage permission to user
+    Given I login as "Administrator"
+    And I have a File document
+    And This document has file "sample.png" for content
+    When I browse to the document
+    And I give writeColdStorage permission to "Jack" on the document
+    Then I can see that "Jack" has the WriteColdStorage permission
+    
+  Scenario: User with WriteColdStorage permission can send file to cold storage
+    Given I login as "Jack"
+    And I have a File document
+    And This document has file "sample.png" for content
+    And I have permission WriteColdStorage for this document
+    When I browse to the document
+    And I send the file to cold storage
+    Then I can see the file is stored in cold storage
+
+  Scenario: User with ReadWrite permission can't send file to cold storage
+    Given I login as "Jack"
+    And I have a File document
+    And This document has file "sample.png" for content
+    And I have permission ReadWrite for this document
+    When I browse to the document
+    Then I cant send the file to cold storage
```

Comment:
I think the scenario is to check the not availability to send content into Cold Storage for a not granted user, I mean a user without a `WriteColdStorage` a given document. But here `Jack` already has the permission that come from the second scenario `User with WriteColdStorage permission can send file to cold storage`

Reply:
The scenarios are independent so Jack doesn't have the WriteColdStorage permission set in 2nd scenario.

Reply:
ah ok, but why Jack  is still able to move into cold storage `Then I cant send the file to cold storage`

Reply:
The step says "I cant send". It's to validate that he can't see the action button.

Reply:
sorry, but it's a little confusing for me. Is it possible to create a step saying `I cannot see the send to cold storage button`. My idea is to be more consistent with the scenario name.

Reply:
Sure

Reply:
Thanks 👍